### PR TITLE
feat(icons): update clock icon for shared link expiry (#1741)

### PR DESCRIPTION
**Pull request by danubevictoria**
BREAKING CHANGE: The clock icon viewbox is now 0x0x16x16 instead of 0x0x15x15. Default icon color is now `#909090`.

Updating clock icon that is used for the Shared Link Expiration Badge.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @danubevictoria has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconClockPreview.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
+const IconClockPreview = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
```
Comment (cweeii) [inline src/icons/general/IconClockPreview.js]: Can you use a variable here?

File: src/icons/general/IconClockPreview.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <AccessibleSVG
+        className={`icon-clock-preview ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 16"
+        width={width}
+    >
+        <g className="fill-color" fill={color}>
```
Comment (priyajeet) [inline src/icons/general/IconClockPreview.js]: Unnecessary g, move props to path

File: src/icons/general/__tests__/IconClockPreview.test.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconClockPreview from '../IconClockPreview';
+
+describe('icons/general/IconClockPreview', () => {
+    const getWrapper = (props = {}) => shallow(<IconClockPreview {...props} />);
+
+    test('should correctly render default icon', () => {
```
Comment (priyajeet) [inline src/icons/general/__tests__/IconClockPreview.test.js]: Probably add a snapshot test too

File: src/icons/general/IconClockPreview.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+
```
Comment (mickr) [inline src/icons/general/IconClockPreview.js]: Do we need the empty lines here?

File: src/icons/general/IconClockPreview.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
```
Comment (mickr) [inline src/icons/general/IconClockPreview.js]: Do we need the empty lines between imports?

File: src/icons/general/IconClockPreview.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+const IconClockPreview = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-clock-preview ${className}`}
```
Comment (mickr) [inline src/icons/general/IconClockPreview.js]: Minor: Do we have the classNames util as to not have an empty space in the class if the default is used?

File: src/icons/general/IconClockPreview.js (hunk 1 above)
Comment (danubevictoria) [reply src/icons/general/IconClockPreview.js]: yes

File: src/icons/general/IconClockPreview.js (hunk 5 above)
Comment (danubevictoria) [reply src/icons/general/IconClockPreview.js]: removed

File: src/icons/general/IconClockPreview.js (hunk 6 above)
Comment (danubevictoria) [reply src/icons/general/IconClockPreview.js]: updated

File: src/icons/general/__tests__/IconClockPreview.test.js (hunk 3 above)
Comment (danubevictoria) [reply src/icons/general/__tests__/IconClockPreview.test.js]: added

File: src/icons/general/IconClock.js [hunk 7]
```diff
@@ -1,22 +1,29 @@
...
 import AccessibleSVG from '../accessible-svg';
-
 import type { Icon } from '../flowTypes';
+import { bdlGray50 } from '../../styles/variables';
 
-const IconClock = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
+const IconClock = ({ className = '', color = bdlGray50, height = 16, title, width = 16 }: Icon) => (
     <AccessibleSVG
-        className={`icon-clock ${className}`}
+        className={classNames('icon-clock', className)}
```
Comment (priyajeet) [inline src/icons/general/IconClock.js]: Classname should follow SUIT style, see other new icon files as an example

Comment (danubevictoria) [reply src/icons/general/IconClock.js]: updated & updated associated scss

**Pull request merged**